main.launch.py

```diff
@@ -56,6 +56,17 @@ def generate_launch_description():
             ("outgoing_can_frames", "outgoing_can_frames")
         ]
     )
+
+    speedometer_reporter = Node(
+        package='can_translation',
+        executable='float_reporter',
+        parameters=[(path.join(param_dir,"interface","speedometer_reporter.param.yaml"))]
+        remappings=[
+            ("incoming_can_frames", "incoming_can_frames_can1"),
```

Comment:
Will we need to change "can1" occasionally between restarts?

Reply:
Yes... in fact, on the car right now, I think it's can0. This is a clumsy solution from a time when I didn't know topic remappings were a thing, and I'm planning to phase it out whenever we start revising the can interface nodes.